parachain/pallets/verifier-lightclient/src/lib.rs

```diff
@@ -23,6 +23,19 @@ mod mock;
 #[cfg(test)]
 mod tests;
 
+#[derive(Clone, PartialEq, RuntimeDebug)]
+pub struct EthashConfiguration {
```

Comment:
Is this configuration struct really necessary? It only defines 1 field and we could define it on the pallet `Trait` instead (this is kinda what is Trait is meant to be used for).

```rust
pub trait Trait: system::Trait {
	type Event: From<Event> + Into<<Self as system::Trait>::Event>;
	/// Ethash PoW configuration
	type VerifyPoW: Get<bool>;
}
```

Reply:
You're right. I based this on parity-bridges-common: https://github.com/paritytech/parity-bridges-common/blob/master/modules/ethereum/src/lib.rs#L375.

Theoretically there are a number of params for Ethash that we might want to make configurable. But we can cross that bridge when we get there. I'll update